gym/wrappers/autoreset.py

```diff
@@ -1,18 +1,20 @@
+"""Wrapper that autoreset environments when `done=True`."""
 import gym
 
 
 class AutoResetWrapper(gym.Wrapper):
-    """
-    A class for providing an automatic reset functionality
-    for gym environments when calling self.step().
-
-    When calling step causes self.env.step() to return done,
-    self.env.reset() is called,
-    and the return format of self.step() is as follows:
+    """A class for providing an automatic reset functionality for gym environments when calling :meth:`self.step()`.
 
-    new_obs, terminal_reward, terminal_done, info
+    When calling step causes :meth:`self.env.step()` to return done, :meth:`self.env.reset` is called,
+    and the return format of :meth:`self.step()` is as follows: `(new_obs, terminal_reward, terminal_done, info)`
+     - new_obs is the first observation after calling :meth:`self.env.reset`
+     - terminal_reward is the reward after calling :meth:`self.env.step`, prior to calling :meth:`self.env.reset`.
+     - terminal_done is always True
+     - info is a dict containing all the keys from the info dict returned by the call to :meth:`self.env.reset`,
+       with an additional key "terminal_observation" containing the observation returned by the last call to :meth:`self.env.step`
+       and "terminal_info" containing the info dict returned by the last call to :meth:`self.env.step`.
 
```

Comment:
Use double backticks. Also, I think that double backticks should be used in the list of bulletpoints (e.g. "- ``terminal_done``is always True")

Reply:
Thanks, updated most of the top docstring as I found several more issues